dds_web/api/schemas/file_schemas.py

```diff
@@ -0,0 +1,93 @@
+"""File related marshmallow schemas."""
+
+####################################################################################################
+# IMPORTS ################################################################################ IMPORTS #
+####################################################################################################
+
+# Standard Library
+from datetime import datetime
+
+# Installed
+import marshmallow
+import sqlalchemy
+
+# Own modules
+from dds_web.database import models
+import dds_web.utils
+from dds_web.api.schemas import project_schemas
+
+####################################################################################################
+# SCHEMAS ################################################################################ SCHEMAS #
+####################################################################################################
+
+
+class NewFileSchema(project_schemas.ProjectRequiredSchema):
+    """Validates and creates a new file object."""
+
+    # Length minimum 1 required, required=True accepts empty string
+    name = marshmallow.fields.String(required=True, validate=marshmallow.validate.Length(min=1))
+    name_in_bucket = marshmallow.fields.String(
```

Comment:
The length validators alone might not be sufficient? Aren't there additional limitations on the filenames allowed in buckets?

Reply:
I haven't been able to find any limitations regarding that. Only the buckets. But it's possible. We haven't run into that issue yet though.